lib/matplotlib/axes/_axes.py

```diff
@@ -6997,6 +6997,8 @@ def hist(self, x, bins=None, range=None, density=False, weights=None,
         (``plt.stairs(*np.histogram(data))``), or by setting *histtype* to
         'step' or 'stepfilled' rather than 'bar' or 'barstacked'.
         """
+        kwargs = cbook.normalize_kwargs(kwargs, mpatches.Patch)
+
         # Avoid shadowing the builtin.
         bin_range = range
         from builtins import range
```

Comment:
It's a minor nitpick, but we really want to 'fix' the `range` import as early as possible. This isn't a lot of code, but it opens the door to accidentally inserting more before it later:
```suggestion
        # Avoid shadowing the builtin.
        bin_range = range
        from builtins import range

        kwargs = cbook.normalize_kwargs(kwargs, mpatches.Patch)
```

Reply:
Looks good to me!